### DISCUSSION TITLE
Open Appstore or Playstore from app

### DISCUSSION THREAD

**Discussion by FM1973** [dotnet/maui#18575] [Q&A] [answered]
Hello.
Can someone guide me into the right direction how to open the Apple Appstore or Google Playstore from within my app?
I manage the version history of my app on the server side and want to tell my users when a new version is available.
Then the user should be able to cklick a button and get to the store wher she/he can update to the newest version.

Thanks for your help!

**Comment by licon4812** [comment, marked as answer]
[Launcher - .NET MAUI](https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/appmodel/launcher?tabs=android)

Link to a play store listing - http://play.google.com/store/apps/details?id=<package_name>

[Apple app store short links Dev Docs](https://developer.apple.com/library/archive/qa/qa1633/_index.html)

**Reply by FM1973** [reply]
Works like a charm, thanks!

**Comment by feed755x** [comment]
For android:

var uri = new Uri($"market://details?id={**packageName**}");
await Browser.Default.OpenAsync(uri, BrowserLaunchMode.External);
